Classes/Systems/GithubClient.swift

```diff
@@ -28,16 +28,34 @@ struct GithubClient {
         let logoutOnAuthFailure: Bool
         let completion: (DataResponse<Any>, Page?) -> Void
 
-        init(
+        static func api(
```

Comment:
Not too sure about this approach with `static func`s. Would it be more readable in another form, like eg. `APIRequest.init`? Or something else entirely? Would love some feedback!

Reply:
Let's have a chat about this in the office :)

Reply:
Yep, we have to get rid of this when / if we pull out `Request`. Then we need another way of injecting the baseURL, but good catch regardless.